README.md

```diff
@@ -69,35 +69,9 @@ npm install bson
 
 ## Documentation
 
-### Objects
-
-<dl>
-<dt><a href="#EJSON">EJSON</a> : <code>object</code></dt>
-<dd></dd>
-</dl>
-
-### Functions
-
-<dl>
-<dt><a href="#setInternalBufferSize">setInternalBufferSize(size)</a></dt>
-<dd><p>Sets the size of the internal serialization buffer.</p>
-</dd>
-<dt><a href="#serialize">serialize(object)</a> ⇒ <code>Buffer</code></dt>
-<dd><p>Serialize a Javascript object.</p>
-</dd>
-<dt><a href="#serializeWithBufferAndIndex">serializeWithBufferAndIndex(object, buffer)</a> ⇒ <code>Number</code></dt>
-<dd><p>Serialize a Javascript object using a predefined Buffer and index into the buffer, useful when pre-allocating the space for serialization.</p>
-</dd>
-<dt><a href="#deserialize">deserialize(buffer)</a> ⇒ <code>Object</code></dt>
-<dd><p>Deserialize data as BSON.</p>
-</dd>
-<dt><a href="#calculateObjectSize">calculateObjectSize(object)</a> ⇒ <code>Number</code></dt>
-<dd><p>Calculate the bson size for a passed in Javascript object.</p>
-</dd>
-<dt><a href="#deserializeStream">deserializeStream(data, startIndex, numberOfDocuments, documents, docStartIndex, [options])</a> ⇒ <code>Number</code></dt>
-<dd><p>Deserialize stream data as BSON documents.</p>
-</dd>
-</dl>
+### BSON 
+
+[API documentation](https://mongodb.github.io/node-mongodb-native/Next/modules/BSON.html)
```

Comment:
Note that we retained the EJSON documentation in the readme since it would require code restructuring to surface the EJSON docs in the API reference. We will file a follow-up ticket to address this issue.

Reply:
thanks! post the ticket here when it's filed TIA

Reply:
Just filed NODE-5252 to track this issue.